src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/ReportPageTests.kt

```diff
@@ -330,7 +330,7 @@ class ReportPageTests : SeleniumTest()
 
         assertThat(listItems[0].findElement(By.cssSelector("span.reader-display-name")).text).isEqualTo("no.perms@example.com")
         assertThat(listItems[0].findElement(By.cssSelector("div")).text).isEqualTo("no.perms@example.com")
-        assertThat(listItems[0].findElements(By.cssSelector("span.remove-reader")).count()).isEqualTo(1)
+        assertThat(listItems[0].findElements(By.cssSelector("span.remove-reader")).count()).isEqualTo(0)
```

Comment:
Super minor but could we change `no.perms@example.com` to be `no.individual.perms@example.com` just to make it easier to follow what's going on here? Then feel free to merge, code all looks good.